drivers/display/ssd16xx.c

```diff
@@ -663,16 +669,21 @@ static int ssd16xx_controller_init(const struct device *dev)
 		}
 	}
 
-	err = ssd16xx_write_uint8(dev, SSD16XX_CMD_DUMMY_LINE,
-				  SSD16XX_VAL_DUMMY_LINE);
-	if (err < 0) {
-		return err;
+	if (config->override_dummy_line) {
+		tmp[0] = config->dummy_line;
+		err = ssd16xx_write_uint8(dev, SSD16XX_CMD_DUMMY_LINE,
```

Comment:
We have in tree boards and shields that use this, you did not change them.

Reply:
Do you know which boards require this? As far as I can tell, the value that was written to this register was incorrect for the majority of the panels.

Reply:
This has now been fixed.